src/main/java/com/celarli/commons/vfs/provider/google/GCSFileNameParser.java

```diff
@@ -0,0 +1,57 @@
+package com.celarli.commons.vfs.provider.google;
+
+import org.apache.commons.vfs2.FileName;
+import org.apache.commons.vfs2.FileSystemException;
+import org.apache.commons.vfs2.FileType;
+import org.apache.commons.vfs2.provider.AbstractFileNameParser;
+import org.apache.commons.vfs2.provider.UriParser;
+import org.apache.commons.vfs2.provider.VfsComponentContext;
+
+
+public class GCSFileNameParser extends AbstractFileNameParser {
+
+    /**
+     * GCS file name parser instance
+     */
+    private static final GCSFileNameParser instance = new GCSFileNameParser();
```

Comment:
Conventionally, as a static final this variable should be named `INSTANCE` in capitals.

Reply:
done